libs/sdk-ui-dashboard/src/model/store/filterContext/filterContextSelectors.ts

```diff
@@ -266,3 +266,62 @@ export const selectAttributeFilterDescendants = createMemoizedSelector((localId:
         return Array.from(result);
     }),
 );
+
+/**
+ * Creates a selector for selecting all filters with different reference than the given one.
+ *
+ * @internal
+ */
+export const selectOtherContextAttributeFilters = createMemoizedSelector((ref?: ObjRef) =>
+    createSelector(selectFilterContextAttributeFilters, (attributeFilters): IDashboardAttributeFilter[] => {
+        return attributeFilters.filter(
+            (attributeFilter) => !areObjRefsEqual(attributeFilter.attributeFilter.displayForm, ref),
+        );
+    }),
+);
+
+/**
+ * Creates a selector for selecting filter identifier to index map.
+ *
+ * @remarks
+ * The mapping's purpose is to enable efficient handling and reading from the connecting attributes
+ * matrix.
+ *
+ * @see {@link selectConnectingAttributesMatrix} for details.
+ *
+ * @internal
+ */
+export const selectFiltersToIndexMap = createSelector(selectSelf, (state) => state.filtersToIndexMap);
+
+/**
+ * Creates a selector for selecting connecting attributes matrix.
+ *
+ * @remarks
+ * The matrix is used to check if the pair of attribute filters are able to create parent child
+ * relationship.
+ *
+ * @internal
+ */
+export const selectConnectingAttributesMatrix = createSelector(
+    selectSelf,
+    (state) => state.connectingAttributeMatrix,
+);
+
+/**
+ * Creates a selector for selecting connecting attributes matrix.
```

Comment:
The TSDoc is not correct (seems copy-pasted from the previous function). Please check it again.

Reply:
Documentation corrected.